plugins/importexport/native/locale/en_US/locale.po

```diff
@@ -125,3 +134,27 @@ msgid "plugins.importexport.native.import.error.publishedDateMissing"
 msgstr ""
 "The article \"{$articleTitle}\" is contained within an issue, but has no "
 "published date."
+
+msgid "plugins.importexport.common.export"
+msgstr "Export"
```

Comment:
Generic notes for the locale keys:
- Check if generic texts already exist on the localization files and re-use them, this one does exist for example as `common.export`.
- In general, try to not create new locale keys on old branches, these are probably not going to be translated (due to problems in how we manage the translations across all the branches, the old branches are currently left to rot).
- If the locale key is going to be re-used in other applications, create them on the `pkp-lib` submodule, then their translation will be shared.